src/main/java/seedu/address/model/TeachingAssistantBuddy.java

```diff
@@ -21,7 +23,7 @@
 
     private final UniqueModuleList modules;
     private final UniqueStudentList students;
-    private final UniqueTaskList tasks;
+    //private final UniqueTaskList tasks;
```

Comment:
a task list should under a module or in a student so can just delete here also?

Reply:
You are right. Will delete.